### PR TITLE
Topic 1

### PR DESCRIPTION
**Pull request by MarcosBrian**
exercise 1, 2, 3

### CODE REVIEW

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		</footer>   
+	</section>
+	<aside>
+		ASIDE
+	</aside>
+	<footer>
+		FOOTER
+	</footer>
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: this scripts are wrong

File: TOPIC-1/exercise 3/CallAPI.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        method: 'GET',
+        url: 'http://api.icndb.com/jokes/random',
+        boolean: 'true'
+    }
+    return callapi(Capi);
+}
+
+
+
+function callapi(Capi){
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 3/CallAPI.js]: Parameter name should be more intuitive.. what do you think?

File: TOPIC-1/exercise 3/CallAPI.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+	if (this.readyState == 4 && this.status == 200){
+		let resp = JSON.parse(this.responseText);
+		let val = resp.value;
+		document.getElementById("response").innerText = val.joke;
+		}
+	else
+		{
+            document.getElementById("response").innerText = call.statusText;
+            document.getElementById("response").style.backgroundColor = "darkred";
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 3/CallAPI.js]: Please Use Single quotes

File: TOPIC-1/exercise 3/CallAPI.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		}
+	};
+		call.onerror = function () {
+        document.getElementById("response").innerText = call.statusText;
+        document.getElementById("response").style.backgroundColor = "darkred";
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 3/CallAPI.js]: same as above

File: TOPIC-1/exercise 4/index.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	</nav>
+	
+	<section>
+			<p>Repositories List:</p>
+			<ul id="list">
+			</ul>
+	</section>
+	<script src="search.js"></script>
+</body>
+</html>
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/index.html]: add empty line after

File: TOPIC-1/exercise 4/search.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+function searching(){
+	let val = search.value;
+	if (val)
+	{
+		let call = {
+			method: 'GET',
+			url: 'https://api.github.com/search/repositories?q=' + val,
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/search.js]: Use template literals

File: TOPIC-1/exercise 4/search.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+			});
+			request.send();
+			return promise;
+		}
+		request(call)
+			.then(resp => {
+				let result = resp.items;
+				for(let n = 0; n < result.length; n++)
+				{
+					document.getElementById("list").innerHTML+="<li><a href=\"https://github.com/"+result[n].full_name+"\">"+result[n].name+"</a></li>";
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/search.js]: Template literals are preferred

File: TOPIC-1/exercise 4/style.css [hunk 8]
```diff
@@ -0,0 +1,19 @@
+label{
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/style.css]: space after curly brace

File: TOPIC-1/exercise 4/style.css [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+    font-weight: bold;
+    background: -webkit-linear-gradient(#f00, #ff0);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    text-shadow: #FF99cc 0.1em 0.1em 0.2em;
+}
+
+
+
+p{
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/style.css]: same as above

File: TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+		<script src="js/main.js"></script>
+
+		<!-- Google Analytics: change UA-XXXXX-Y to be your site's ID. -->
+		<script>
+			window.ga=function(){ga.q.push(arguments)};ga.q=[];ga.l=+new Date;
+			ga('create','UA-XXXXX-Y','auto');ga('send','pageview')
+		</script>
+		<script src="https://www.google-analytics.com/analytics.js" async defer></script>
+	</body>
+</html>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html]: Add empty line

File: TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+		</footer>   
+	</section>
+	<aside>
+		ASIDE
+	</aside>
+	<footer>
+		FOOTER
+	</footer>
+		
+		<script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
```
Comment (DemianMoschin-GLB) [inline TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html]: This is not right

File: TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html (hunk 1 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-2/exercise 2-1/exercise 2-1.html]: resolved

File: TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html (hunk 11 above)
Comment (MarcosBrian) [reply TOPIC-0-EXERCISE-2/exercise 2-2/exercise 2-2.html]: corrected

File: TOPIC-1/exercise 3/CallAPI.js (hunk 2 above)
Comment (MarcosBrian) [reply TOPIC-1/exercise 3/CallAPI.js]: renamed

File: TOPIC-1/exercise 4/style.css (hunk 8 above)
Comment (MarcosBrian) [reply TOPIC-1/exercise 4/style.css]: spaces added

File: TOPIC-1/exercise 4/style.css (hunk 9 above)
Comment (MarcosBrian) [reply TOPIC-1/exercise 4/style.css]: spaces added

File: TOPIC-1/exercise 4/search.js [hunk 12]
```diff
@@ -0,0 +1,51 @@
+function searching()
+{
+	let searchBtn = document.getElementById('search');
+	let search = searchBtn.value;
+	if (search) 
+	{
+		let config = {
+			method: 'GET',
+			url: 'https://api.github.com/search/repositories?q=' + search,
```
Comment (DemianMoschin-GLB) [inline TOPIC-1/exercise 4/search.js]: Template literals please

**Pull request merged**